frontend/packages/ceph-storage-plugin/src/components/volume-snapshot/volume-snapshot.tsx

```diff
@@ -151,16 +145,15 @@ export const VolumeSnapshotTableRow: RowFunction<K8sResourceKind> = ({
       />
     </TableData>
     <TableData className={snapshotTableColumnClasses[1]}>
-      {obj?.metadata?.creationTimestamp}
+      <Timestamp timestamp={obj.metadata.creationTimestamp} />
     </TableData>
     <TableData className={snapshotTableColumnClasses[2]}>
-      <Status status={obj?.status?.readyToUse ? 'Ready' : 'Not Ready'} />
+      <Status status={obj.status.readyToUse ? 'Ready' : 'Not Ready'} />
     </TableData>
     <TableData className={snapshotTableColumnClasses[3]}>
-      {obj?.status?.restoreSize || 'No Data'}
+      {obj.status.restoreSize || 'No Data'}
```

Comment:
In console if I am not wrong, if some data is not available then a dash is used `-` for lists.